main.rb

```diff
@@ -20,7 +21,7 @@ def inputs(int)
     end
   end
 
-  def starting
+  def start
     puts 'Please choose an option by entering a number'
     puts '1 - List all books'
     puts '2 - List all people'
```

Comment:
- [ ] Kindly notice that the program has to continue until the user chooses to close it for that you need to repeat showing the options to the user after every operation.

```ruby
def start
    num = 0
    while(num != 7)
      puts 'Please choose an option by entering a number'
      puts '1 - List all books'
      puts '2 - List all people'
      puts '3 - Create a person'
      puts '4 - Create a book'
      puts '5 - Create a rental'
      puts '6 - List all rentals for a given id'
      puts '7 - Exit'
      num = gets.chomp
      inputs(num)
    end
end
```